notes/stories/videogame/videogame-final-fantasy.st.txt

```diff
@@ -131,6 +131,73 @@ European dragon [Firion, Maria, and Guy saved a race of wyverns, known as the Dr
 what if I were swallowed by an enormous beast [The party was swallowed by a leviathan and had to navigate their way out through its intestinal tract.]
 
 
+videogame: Final Fantasy III (1990)
+===================================
+
+:: Title
+Final Fantasy III
+
+:: Date
+1990-04-29
+
+:: Description
+The story revolves around four orphaned youths drawn to a crystal of light.
+The crystal grants them some of its power, and instructs them to go forth and
+restore balance to the world. Not knowing what to make of the crystal's
+pronouncements, but nonetheless recognizing the importance of its words, the
+four inform their adoptive families of their mission and set out to explore
+and bring back balance to the world.
+
+:: Authors
+Square
+
+:: Variation
+NES
+
+:: Ratings
+4 <paul>
+
+:: Genre
+fantasy
+
+:: References
+https://en.wikipedia.org/wiki/Final_Fantasy_III
+
+:: Choice Themes
+good vs. evil [It was up to the Light Warriors to prevent the evil wizard Xande from covering the world in a veil of darkness, and later preventing the malevolent cosmic entity the Cloud of Darkness from reducing the world to a void of nothingness.]
+
+:: Major Themes
+speculative habitat [In a mid story revelation, the Light Warriors discover that the world as they knew it was but a floating continent that hovered high above a much more vast world.]
+crackpot theory [It was revealed that the floating continent had been engineered by a technologically advanced, ancient civilization through the use some crystal-based technology. Although the story is set on an unnamed planet, the idea here is similar to the fringe theory that Atlantis existed and had developed advanced technology. The crackpot theory themes applies for this reason.]
+cosmic balance [It was up to the Light Warriors to return the light and dark worlds into balance. In the end, it was revealed that a malevolent cosmic entity, called the Cloud of Darkness, was attempting to nullify all creation by mixing the light and dark worlds into one (i.e. the idea being that the mixing together of worlds constitutes an imbalance of sorts).]
+supernatural existential risk to civilization [The story culminates with the Light Warriors having to stop the malevolent cosmic entity the Could of Darkness from turning the world into a void of nothingness.]
+hope [The story pointedly concludes with the message that the Light Warriors had gotten their power from the hope of the people, and that hope is the one thing capable of defeating the malevolent cosmic entity the Cloud of Darkness.]
+the supernatural [The use of magical spells is a key component of the game play and story plot.]
```

Comment:
We typically use https://www.themeontology.org/theme.php?name=magic%20powers for this sort of thing (not saying the supernatural might not also have been topical - we haven't quite clarified how to use it)

Reply:
Replaced **the supernatural** by **magic powers**. We'll have to get around to sorting out how to use the magic/supernatural themes one of these days.